client/my-sites/email/email-management/email-home.jsx

```diff
@@ -119,16 +126,20 @@ class EmailManagementHome extends React.Component {
 
 		return this.renderContentWithHeader(
 			<>
-				<EmailListActive
-					domains={ domainsWithEmail }
-					selectedSiteSlug={ selectedSite.slug }
-					currentRoute={ currentRoute }
-					selectedSiteId={ selectedSiteId }
-				/>
+				{ ! showActiveDomainList && (
```

Comment:
This should be:
```suggestion
				{ showActiveDomainList && (
```

You could also set a default prop value of `true`